docs/docusaurus.config.js

```diff
@@ -210,18 +210,18 @@ module.exports = {
       ({
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
-          editUrl: 'https://github.com/woodpecker-ci/woodpecker/edit/master/docs/',
+          editUrl: 'https://github.com/woodpecker-ci/woodpecker/edit/main/docs/',
           includeCurrentVersion: true,
           lastVersion: '1.0',
           versions: {
-            'current': {
+            current: {
               label: 'Next',
               banner: 'unreleased',
             },
             '1.0': {
               label: '1.0.x',
             },
-            '0.15': {
```

Comment:
Intentional?

Reply:
Prettier did this. Maybe I can add it as check to the CI as well 🤔

Reply:
well I did add it as `1.0` will be interpreded as float not string witch will crash docusaurus

but the others are fine :laughing: 

in any case didnt we already enforce formating ?